### PR TITLE
Make ourselves compatible with the new OpenID compliant mode of Auth0

### LINKED ISSUE #83: [auth0] Move auth0 to oidc-conformant mode
Per jabba we need to make this change so that the sso dashboard will be in oidc-conformant mode, per conversation with jabba:

```
10:43 <jabba> ok, so here's what you need to do:
10:43 <jabba> change: Require claim groups:
10:43 <jabba> to
10:43 <jabba> Require claim https://sso.mozilla.com/claim/groups:
```

So as an example we will need to change

```
Require claim groups:<example group>
to
Require claim https://sso.mozilla.com/claim/groups:<example group>
```

And also do this

```
10:43 <jabba> and remove Require claim multifactor:duo
10:44 <jabba> that won't work, and as of 6am tomorrow, all auth will require MFA, regardless
```

### PR DESCRIPTION
**Pull request by gozer**
Fully backwards-compatible

Fixes #83

### CODE REVIEW

File: nubis/files/confd/templates/sso-require-macros.conf.tmpl [hunk 1]
```diff
@@ -1,24 +1,26 @@
...
-    Require claim multifactor:duo
-    <RequireAny>
-{{ if exists "/config/SudoGroups" }}{{ range split (getv "/config/SudoGroups") "," }}      Require claim groups:{{.}}
-{{ end }}{{ end }}
-    </RequireAny>
-  </RequireAll>
+  <RequireAny>
+{{- if exists "/config/SudoGroups" }}{{ range split (getv "/config/SudoGroups") "," }}
+    Require claim groups:{{.}}
+    Require claim https://sso.mozilla.com/claim/groups:{{.}}
```
Comment (limed) [inline nubis/files/confd/templates/sso-require-macros.conf.tmpl:6]: Shouldn't we make the claim URL a configurable option?

Comment (limed) [reply nubis/files/confd/templates/sso-require-macros.conf.tmpl:6]: Not a big deal but I'm ok with these changes

Comment (gozer) [reply nubis/files/confd/templates/sso-require-macros.conf.tmpl:6]: Hrm, I believe you do have a very good point there.

Will do and file a separate PR for that.

Comment (limed) [reply nubis/files/confd/templates/sso-require-macros.conf.tmpl:6]: For now I'll r+ this

**Review by limed — APPROVED** [review]
r+

**Pull request merged**